osu.Game/Configuration/SessionStatics.cs

```diff
@@ -20,6 +20,14 @@ protected override void InitialiseDefaults()
             SetDefault(Static.LastHoverSoundPlaybackTime, (double?)null);
             SetDefault<APISeasonalBackgrounds>(Static.SeasonalBackgrounds, null);
         }
+
+        public void ResetValues()
+        {
+            SetValue(Static.LoginOverlayDisplayed, false);
```

Comment:
I *think* you can get away with

```csharp
ConfigStore.Clear();
InitialiseDefaults();
```

Will need to be tested.

Reply:
Unresolving this, since this hasn't actually been addressed yet.

I'd like to think that doing something like

```csharp
GetOriginalBindable<bool>(Static.LoginOverlayDisplayed).SetDefault();
```

should work, right? Still not perfect, but it compiles at least. `Get<>()` and `GetBindable<>()` return `Bindable`s and not the interface variants...?

Reply:
That looks much better.

The best way to do this would be to iterate through the `Static`enum and set the defaults for each member, but I'm not quite sure how I would get the type of value from the `Static` member to do so.

Reply:
I don't believe the lookup-to-bindable-type mapping is exposed or stored anywhere, no.

Reply:
Okay, in that case we can still improve this further by storing that ourselves in a dictionary. Then in `InitialiseDefaults()` we can use a method like this in-place of `SetDefault()`:

```csharp
void AddDefault<TValue>(Static lookup, TValue value) {
    SetDefault(lookup, value);
    defaultTypes.Add(typeof(TValue), lookup));
}
```

Then in `ResetValues` loop through that list and apply the changes accordingly.

Sound good?

Reply:
Not sure. If this is to be added it should probably be added to the framework mechanism rather than locally here.

Reply:
Fair enough. I'll leave it as is and resolve this for now.